examples/generative/stylegan.py

```diff
@@ -0,0 +1,736 @@
+"""
+Title: Face Generation and Mixing with StyleGAN
+Author: Soon-Yau Cheong
+Date created: 2021/06/27
+Last modified: 2021/06/27
+Description: Implementation of StyleGAN with custom train step.
+"""
+"""
+# StyleGAN
+
+
+This [StyleGAN](https://arxiv.org/abs/1812.04948) implementation is based on the book
+[Hands-on Image Generation with TensorFlow](https://www.amazon.com/dp/1838826785). The
+code from the book's [Github
+repository](https://github.com/PacktPublishing/Hands-On-Image-Generation-with-TensorFlow-2
+.0/tree/master/Chapter07) is refactored into custom train step to enable faster training
+time in non-eager mode. The key features of StyleGAN is to grow the resolution
+progressively and to incorporate style features in the generative process.
+"""
+
+"""
+## Import modules
+"""
+
+import os
+import random
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+from enum import Enum
+from glob import glob
+from functools import partial
+
+import tensorflow as tf
+from tensorflow.keras.layers import Layer, Dense, Input, LeakyReLU, Reshape
```

Comment:
Just import the following:

`from tensorflow import keras`, `from tensorflow.keras import layers`

then you can access objects either via `layers.X` (e.g. `layers.Dense`) or `keras.X` (e.g. `keras.callbacks. ModelCheckpoint`). No need to import individual layers.